tests/Microsoft.Identity.Test.Unit.netcore/Microsoft.Identity.Test.Unit.netcore.csproj

```diff
@@ -7,17 +7,21 @@
     <AssemblyOriginatorKeyFile>../../build/MSAL.snk</AssemblyOriginatorKeyFile>
     <CodeAnalysisRuleSet>../../build/SolutionWideAnalyzerConfig.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
-  
+
   <ItemGroup>
 
     <Compile Include="..\Microsoft.Identity.Test.Unit.net45\**\*.cs" Exclude="..\Microsoft.Identity.Test.Unit.net45\obj\**\*;..\Microsoft.Identity.Test.Unit.net45\bin\**\*" />
-    
+    <Compile Include="../Microsoft.Identity.Test.Core.UIAutomation/UserInformationFieldIds.cs" LinkBase="Integration"></Compile>
```

Comment:
Can we not reference the assembly instead of including the constants from a separate project?  Doing project-level magic can be difficult to track.  I'd rather have the code live in one place instead of getting project-included from other projects.

Reply:
I will add a todo here. I was planning to use project references but this will lead to quite a lot of work which is not really needed here. I plan to create 2 new assemblies for headless E2E tests when we get to write more of them and will do the work then.

---
In reply to: [241281624](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/718#discussion_r241281624) [](ancestors = 241281624)